### ISSUE TITLE
[css-font-loading] FontFace constructor section 2.1 need clarifying

### ISSUE THREAD

**Issue by nmoucht** [w3c/csswg-drafts#4771]
https://drafts.csswg.org/css-font-loading/#dom-fontface-fontface

Section 2.1: 

"If any of them fail to parse correctly, reject font face’s [[FontStatusPromise]] with a DOMException named "SyntaxError", set font face’s corresponding attributes to the empty string, and set font face’s status attribute to "error". Otherwise, set font face’s corresponding attributes to the serialization of the parsed values."

This section should specify when there is a parsing failure, if the all of the font face's attributes should be empty string, or just those that failed to parse.

**Comment by litherum** [comment]
@tabatkins

Gecko and Blink agree that all the attributes get set to the empty string, so we've modified WebKit to match.

Now that all browsers match, the spec should be changed to reflect the agreement between all the browsers.

**Comment by tabatkins** [comment]
I don't think that was my original intention, but the spec is unclear, and it also doesn't really matter. So yes, I agree the spec should match impls here.

However, note:

> Currently Firefox and Chrome seem to set all attributes to some default string when there is a parsing error,

which I've confirmed is true. Does Safari do that, @litherum, or set to an empty string?

**Comment by litherum** [comment]
Oh, I said the wrong thing. We now set all attributes to a default value.

Sorry 😅
